manifests/010-prometheus-rules.yaml

```
@@ -41,3 +41,11 @@ spec:
       annotations:
         message: |
           Samples operator cannot find credentials for 'registry.redhat.io' in the samples pull secret in the openshift namespace.  Unless you have updated the 'samplesRegistry' field in the samples operator config to change which registry the sample ImageStreams are pulled from, you will have failed imports for many of the sample ImageStreams in this case.
+    - alert: SamplesTBRInaccessibleOnBoot
```

nit - do we need to say "OnBoot"?:
```suggestion
    - alert: SamplesTBRInaccessible
```

---

Yeah the check itself is only done on initial startup @adambkaplan 

So I thought the name should reflect that.

And at this time I don't see the need to do the check on a periodic basis.